src/main/scala/it/pagopa/interop/purposeprocess/api/impl/PurposeApiServiceImpl.scala

```diff
@@ -489,6 +493,37 @@ final case class PurposeApiServiceImpl(
       onComplete(result) { clonePurposeResponse[Purpose](operationLabel)(clonePurpose200) }
     }
 
+  override def retrieveLatestRiskAnalysisConfiguration()(implicit
+    contexts: Seq[(String, String)],
+    toEntityMarshallerRiskAnalysisFormConfigResponse: ToEntityMarshaller[RiskAnalysisFormConfigResponse],
+    toEntityMarshallerProblem: ToEntityMarshaller[Problem]
+  ): Route = authorize(ADMIN_ROLE) {
+    val operationLabel = s"Retrieve latest risk analysis configuration"
+    logger.info(operationLabel)
+
+    val result: Future[RiskAnalysisFormConfigResponse] = for {
+      organizationId                   <- getOrganizationIdFutureUUID(contexts)
+      tenant                           <- tenantManagementService.getTenant(organizationId)
+      tenantKind                       <- tenant.kind.fold(
+        purposeVersionActivation.getTenantKindLoadingCertifiedAttributes(tenant.attributes, tenant.externalId)
+      )(Future.successful(_))
+      kindConfig                       <- riskAnalysisServiceSupplier
+        .get()
```

Comment:
⚠️ you are creating a new instance for each request.
Please use the current approach, where the `*Impl` instance is passed as dependency to `PurposeApiServiceImpl` constructor